### ISSUE TITLE
License incompatability: mixes GPL 2 only plus GPL 3 codes

### ISSUE THREAD

**Issue by mr-c** [statnet/network#23]
While preparing this module for Debian, it [was noticed](https://alioth-lists.debian.net/pipermail/r-pkg-team/2020-April/011150.html) that there is an incompatible mixture of GPL-2 and GPL-3 code:

```
Files: R/as.edgelist.R
Copyright: 2003-2015 Statnet Commons
License: GPL-3

Files: inst/network.api/networkapi.c
 inst/network.api/networkapi.h
 src/Rinit.c
 src/access.c
 src/access.h
 src/constructors.c
 src/constructors.h
 src/layout.c
 src/layout.h
 src/utils.c
 src/utils.h
Copyright: Carter T. Butts <buttsc@uci.edu>
           Jeffrey Horner <jeffrey.horner@gmail.com>
License: GPL-2
```

As per https://www.gnu.org/licenses/gpl-faq.en.html#AllCompatibility it is not permissible to mix GPL-2 and GPL-3 code. 

Please consider working with the copyright holders to re-license into a compatible combination. Perhaps everything under GPL-3 or GPL-3+ would be simplest.

Thanks!

**Comment by mr-c** [comment]
To be clear, until this incomparability is resolved, Debian can not package and release this module or any module that depends on it

**Comment by mbojan** [comment]
Thanks! Calling upon @CarterButts @martinamorris @krivit

**Comment by mbojan** [comment]
d6a2a50b84db7bcd2ed755cb3b88020de6b86eff

**Comment by mr-c** [comment]
Thanks!

There are two more files that still need updating:

GPL 2.0 only:
https://github.com/statnet/network/blob/master/inst/network.api/networkapi.h and .c

**Comment by krivit** [comment]
@mr-c, addressed in a7575e2d57b1baadd53b4f98b02935545505ee62 , by the looks of it.

**Comment by mr-c** [comment]
@krivit Can you make a new release so that the relicensing is clear?

**Comment by mr-c** [comment]
(Many thanks, by the way!)

**Comment by krivit** [comment]
@mr-c , fairly soon, I think. @CarterButts ?

**Comment by CarterButts** [comment]
Argh, that's my mistake - I just pushed 1.16.1 and missed those two files.  They are now fixed in that branch on GH, so if another bugfix release is pushed to CRAN from the 1.16.* line it will get the fixes.  They are also fixed in the 1.17 line, so the next release will contain all of those changes.  Sorry about that!
